tx-parser/src/lib.rs

```diff
@@ -13,12 +13,17 @@
 extern crate edn;
 extern crate combine;
 extern crate mentat_tx;
+extern crate mentat_parser_utils;
```

Comment:
This is also likely to require a macro_use pragma if you plan on using the macros there. I'd be surprised if not.